### ISSUE TITLE
All Pages - Missing ALT Text for Agency Logos/Seals

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#8]
While testing using the DHS Trusted Tester process, I found there is no text/ALT text for the four logos at the bottom of page to tell screen reader user which agency they are going to go to.

The ALT text should be from left to right on the logos/seals:

1. United States Department of Labor
2. National Labor Relations Board
3. United States Department of Justice
4. United States Equal Employment Opportunity Commission

Section 508 - 22(a)
WCAG 2.0 - 2.2 Web: Images

**Comment by jfc3-dol** [comment]
Here is a screenshot showing the four agency logos/seals missing ALT text.

![worker dot gov - agency logos or seals missing alt text](https://cloud.githubusercontent.com/assets/10423487/19775296/f7a2c218-9c3d-11e6-9c7d-d4a4a7adab04.png)

**Comment by adamsfallen** [comment]
Updated - 13e47b726d2dbb90396c0e41b002c893655f6637

**Comment by jfc3-dol** [comment]
The updates look good to me.
